stackslib/src/chainstate/nakamoto/tenure.rs

```diff
@@ -509,6 +509,21 @@ impl NakamotoChainState {
         }
     }
 
+    /// Get a nakamoto tenure-change by its tenure ID consensus hash.
+    /// Get the highest such record.
```

Comment:
This also looks like its filtering by tenure changes caused by sortitions -- can you add that to the rustdoc?

Reply:
Sure